addons/website_blog/views/website_blog_templates.xml

```diff
@@ -230,28 +222,28 @@ list of filtered posts (by date or tag).
             </div>
         </section>
 
-        <section id="o_wblog_post_main" t-attf-class="container pt-4 pb-5 #{'anim' in request.params and 'o_wblog_post_main_transition'}">
+        <div id="o_wblog_post_main" t-attf-class="pb-5 #{'anim' in request.params and 'o_wblog_post_main_transition'}">
             <!-- Sidebar-enabled Layout -->
-            <div t-if="opt_blog_post_sidebar" t-attf-class="mx-auto #{opt_blog_post_readable and 'o_wblog_read_with_sidebar'}">
-                <div t-attf-class="d-flex flex-column flex-lg-row #{opt_blog_post_readable and 'justify-content-between'}">
-                    <div id="o_wblog_post_content" t-attf-class="#{opt_blog_post_readable and 'o_container_small mx-0 w-100 flex-shrink-0' or 'w-lg-75'}">
+            <div t-if="opt_blog_post_sidebar" class="mx-auto">
+                <div class="d-flex flex-column flex-lg-row justify-content-start">
+                    <div id="o_wblog_post_content" class="col-lg-9 col-12 px-0 mx-0 w-100 flex-shrink-0">
```

Comment:
Columns must in rows, so I don't know how it must be but this structure here with a column without a row is not valid.

Reply:
`d-flex` is making it a row

Reply:
Sure... bootstrap just created two classes doing the same stuff...

Reply:
I don't need to define the margin 
<img width="387" alt="Capture d’écran 2021-06-30 à 12 40 04" src="https://user-images.githubusercontent.com/18035234/123947141-5345a500-d9a0-11eb-85c7-df1f9f0ff242.png">